admin/client/src/boot/index.js

```diff
@@ -68,6 +68,10 @@ function appBoot() {
   // Bootstrap routing
   const routes = new BootRoutes(store);
   routes.start(window.location.pathname);
+
+  // @TODO - Remove once we remove entwine
+  // Enable top-level css selectors for react-dependant entwine sections
+  window.jQuery('body').addClass('react-boot');
```

Comment:
I don't think it's necessary for this instance, seems to act more like a modernizr for react booting.

Although a check for `window.jQuery` existing would be nice.